modules/bastion/scripts/oracle.template.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+# Copyright 2017, 2019, Oracle Corporation and/or affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl
+
```

Comment:
Could you pls add inline comments to explain what each block of these hardening commands does? It'd be good to surface this info in the solution guide as well.

Reply:
These are based on the CIS benchmark docs. I'll add the the link in the script

Reply:
Won't be using these, instead we'll be recommending to use the CIS image in the OCI Market place